lib/utils/objects.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Return an object filtering out keys that point to undefined values.
+ */
+export const definedValues = <T extends object>(obj: T): T =>
```

Comment:
name doesn't express the intent, perhaps `withOnlyDefinedValues` or `withoutUndefinedValues` would be more intuitive